packages/api/test/codegen/languages/typescript.test.ts

```diff
@@ -1,7 +1,7 @@
 import type { TSGeneratorOptions } from '../../../src/codegen/languages/typescript';
 
-import { promises as fs } from 'fs';
```

Comment:
no idea why i loaded `fs/promises` like this. the things you find in small refactors...